docs/epayment.v1.yml

```
@@ -234,7 +199,7 @@ components:
       type: object
       description: |-
         Can be one of:
-          * The user subscription identifier received in `$.profile.sub` or from the Vipps Login API
+          * The user subscription identifier received in the `$.profile.sub` property from the response body or from the Vipps Login API
```

The `$` syntax is not needed for plain JSON, I think? 
```suggestion
          * The user subscription identifier received in the `profile.sub` property from the response body or from the Vipps Login API
```